modules/oe_list_pages_open_vocabularies/tests/src/Kernel/ListPagesSearchApiConfiguratorTest.php

```diff
@@ -161,14 +161,14 @@ public function testVocabularyAssociationChanges(): void {
     $facet_1 = Facet::load($id_1);
     $this->assertEquals('New label', $facet_1->label());
     $this->assertEquals($field_id, $facet_1->getFieldIdentifier());
-    $this->assertEqual(SearchApiConfigurator::MAX_WEIGHT, $facet_1->getWeight());
+    $this->assertEquals(SearchApiConfigurator::MAX_WEIGHT, $facet_1->getWeight());
```

Comment:
Looks like also you have to swap the order of arguments (see the signature difference of method).

Reply:
Change updated on next update.